### ISSUE TITLE
(Question) winit support

### ISSUE THREAD

**Issue by hardBSDk** [ratatui-org/crates-tui#48]
Does this program use winit? because I get a `wayland-backend` crate error in a system that don't support Wayland yet.

winit or some GUI toolkit would fix it.

**Comment by joshka** [comment]
It shouldn't have any Wayland specific code (being a console based tui app and not a gui). It's possible that there's a system specific dependency in the graph.

Please provide more details on the build failure and a copy of the output of cargo tree. Thanks

**Comment by hardBSDk** [comment]
@joshka Yes, sorry for my mistake, it uses Ratatui for the TUI interface.

This is the dependency using Wayland:

```
── copypasta v0.10.1
│   ├── smithay-clipboard v0.7.0
│   │   ├── libc v0.2.153
│   │   ├── smithay-client-toolkit v0.18.1
```
